pkg/mountutil/mountutil_linux.go

```diff
@@ -103,13 +103,17 @@ func parseVolumeOptionsWithMountInfo(vType, src, optsRaw string, getMountInfoFun
 	var (
 		writeModeRawOpts   []string
 		propagationRawOpts []string
+		bindOpts           []string
 	)
 	for _, opt := range strings.Split(optsRaw, ",") {
 		switch opt {
 		case "rw", "ro", "rro":
 			writeModeRawOpts = append(writeModeRawOpts, opt)
 		case "private", "rprivate", "shared", "rshared", "slave", "rslave":
 			propagationRawOpts = append(propagationRawOpts, opt)
+		case "bind", "rbind":
+			// bind means not recursively bind-mounted, rbind is the opposite
+			bindOpts = append(bindOpts, opt)
```

Comment:
This seems to add `bind` and `rbind` to `-v` option. Is it expected change? If so, docs should be added to README.

```console
# nerdctl run --rm -it -v /tmp/testmnt/rw:/mnt1:bind alpine /bin/sh
/ # apk add findmnt -q
/ # findmnt -nR /mnt1
/mnt1  tmpfs[/testmnt/rw] tmpfs  rw,nosuid,nodev,relatime,mode=777,inode64
```

Reply:
Added.

Reply:
I'm not sure that's a good idea. We should keep docker consistency. 
We can keep it like this now and refacto in a next PR